modules/client-configuration/pages/autoinst-profiles.adoc

```diff
@@ -61,31 +70,162 @@ This can overwrite your customization.
+One of the possibilities offered by the {kickstart} profiles is the file preservation lists. If you have many custom configuration files located on a client you want to {kickstart}, you can save them as a list, and associate that list with the {kickstart} profile to be used.
 
+.Procedure: Create a File Preservation List
+. In the {productname} {webui}, navigate to menu:Systems[Autoinstallation > File Preservation] and click btn:[Create File Preservation List].
+. Enter a suitable label, and list absolute paths to all files and directories you want to save.
+. Click btn:[Create List].
+. Include the file preservation list in your {kickstart} profile by navigating to menu:Systems[Autoinstallation > Profiles] and selecting the profile you want to edit. Navigate to the menu:System Details[File Preservation] subtab and select the file preservation list to include.
 
 [NOTE]
 ====
 File preservation lists are limited to a total size of 1{nbsp}MB.
-Special devices like [path]``/dev/hda1`` and [path]``/dev/sda1`` are not supported.
+Special devices like [path]``/dev/hda1`` and [path]``/dev/sda1`` cannot be preserved.
 Only use file and directory names, you cannot use regular expression wildcards.
 ====
+
+For more information about Kickstart, see the Red Hat documentation.
+
+
+[[templates-syntax]]
+== Templates Syntax
+
+Parts of your installation file will be replaced during the installation:
+
+* variables will be replaced with single values;
+* code snippets will be replaced with whole sections of text;
+* escaped symbols or sections will not be replaced.
+
+You can create autoinstallation variables and code snippets within the {productname} {webui}.
+
+This mechanism allows you to reinstall large numbers of systems, without having to manually create profiles for each of them.
+
+It's a template engine called Cheetah that allows Cobbler to do these replacements.
+
+Within a profile, the [guimenu]``Autoinstallation File`` tab allows you to see the result of the substitutions.
+
+For instructions and details about:
+* variables, see xref:client-configuration:autoinst-profiles#variables[Variables];
+* code snippets, see xref:client-configuration:autoinst-profiles#code-snippets[Code Snippets];
+* escaping, see xref:client-configuration:autoinst-profiles#variables[Escaping].
+
+
+[[variables]]
+=== Variables
+
+Autoinstallation variables can be used to substitute values into {kickstart} and {ay} profiles.
+
+To define a variable, from the profile, navigate to the [guimenu]``Variables`` subtab, and create a name-value pair ([replaceable]``name=value``) in the text box.
+
+For example, if you want to autoinstall a system that joins the network of a specified organization (for example the Engineering department), you can create a profile variable to set the IP address and the gateway server address to a variable that any system using that profile uses.
+
```

Comment:
```suggestion
```

Reply:
reorganized

Reply:
not 100% sure because of the many renumbering of the lines, but trying